README.md

```diff
@@ -65,6 +65,24 @@ $ gsutil cat gs://my-bucket/logs/output
 
 The script file format is described in the [source code for the command][3].
 
+### Using gcsfuse with the pipelines tool
+
+The `pipelines` tool can use [gcsfuse][gcs-fuse] to localize input files
+instead of copying them one by one with `gsutil`.  There are a few important
+caveats when using the `--fuse` flag:
```

Comment:
I think this flag should be called `--gcsfuse`.

Reply:
I considered that but figured it was better to say '--fuse' in the event that we at some point support _other_ localization schemes using fuse.  Can change if you feel strongly (though I imagine it should be --gcs-fuse?)

Reply:
Alright, makes sense to me.